test/e2e/util.go

```diff
@@ -58,6 +61,19 @@ func clusterIPService(name, namespace string) *corev1.Service {
 	return testService(name, namespace, corev1.ServiceTypeClusterIP)
 }
 
+func testRoute(name, namespace string) *routev1.Route {
+	return &routev1.Route{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace:   namespace,
+			Name:        name,
+			Annotations: map[string]string{},
+		},
+		Spec: routev1.RouteSpec{
+			Host: "my-domain.com",
```

Comment:
This can be passed as parameter too. Typically the hosts of routes are of the format: `<given-hostname>.<base-cluster-domain>`. In the test you can pass the host which is made of any test name + the domain of just created zone.